src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html

```diff
@@ -22,7 +22,9 @@
     <link rel="import" href="/sys/starcounter.html">
     <link rel="import" href="/sys/starcounter-include/starcounter-include.html">
 
-    <link rel="import" href="/sys/bootstrap.html">
+    <link rel="stylesheet" href="/sys/underwear.css/underwear.css">
+    <link rel="stylesheet" href="/sys/uniform.css/uniform.css">
+    
     <style>
```

Comment:
Are the lines https://github.com/Starcounter/StarcounterClientFiles/blob/94d187633b43b2c645450bee6536673b79276085/src/StarcounterClientFiles/wwwroot/sys/app-shell/app-shell.html#L28-L35 still needed?

Reply:
Yes, because we'll remove this part from `underwear.css` as per @tomalec comments here (https://github.com/Starcounter/Underwear.css/pull/14#discussion_r162818786)